### PR TITLE
BLADE-611 URLClassLoader keep jar file openning

### PR DESCRIPTION
**Pull request by simonjhy**

### CODE REVIEW

File: cli/src/main/java/com/liferay/blade/cli/command/CreateCommand.java [hunk 1]
```diff
@@ -262,8 +267,6 @@ else if (isDefaultModulesDirSet) {
 			if (m != null) {
 				m.setAccessible(false);
 			}
-
-			thread.setContextClassLoader(oldContextClassLoader);
```
Comment (gamerson) [inline cli/src/main/java/com/liferay/blade/cli/command/CreateCommand.java:266]: hey @Seiphon  why do we not need to re-set the thread context classloader to the old classloader?  Does this not break the classloader afterwards?

**Comment by gamerson** [comment]
hey @Seiphon  on java11, when I run blade, I see this message:
```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.liferay.blade.cli.util.CombinedClassLoader (file:/Users/greg/my%20repos/liferay-blade-cli/cli/build/libs/blade.jar) to field java.net.URLClassLoader.ucp
WARNING: Please consider reporting this to the maintainers of com.liferay.blade.cli.util.CombinedClassLoader
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

So we will not be able to use this method fo solving this problem.

File: cli/src/main/java/com/liferay/blade/cli/command/CreateCommand.java (hunk 1 above)
Comment (Seiphon) [reply cli/src/main/java/com/liferay/blade/cli/command/CreateCommand.java:266]: Actually, when execute method finished , In here https://github.com/liferay/liferay-blade-cli/blob/5108d3b0c897950f0a06181ffcc50f60cbacc949/cli/src/main/java/com/liferay/blade/cli/BladeCLI.java#L1328
,the classloader will be re-set to the old one, so I think we don't need to reset it twice.

**Comment by Seiphon** [comment]
> hey @Seiphon on java11, when I run blade, I see this message:
> 
> ```
> WARNING: An illegal reflective access operation has occurred
> WARNING: Illegal reflective access by com.liferay.blade.cli.util.CombinedClassLoader (file:/Users/greg/my%20repos/liferay-blade-cli/cli/build/libs/blade.jar) to field java.net.URLClassLoader.ucp
> WARNING: Please consider reporting this to the maintainers of com.liferay.blade.cli.util.CombinedClassLoader
> WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
> WARNING: All illegal access operations will be denied in a future release
> ```
> 
> So we will not be able to use this method fo solving this problem.

Since this method can not solving this problem, I will try anothoer way, still working on it.

**Comment by gamerson** [comment]
hey @Seiphon  I discussed an alternate approach with @simonjhy  yesterday, so closing this now.  Thanks for your great work here, but i'm worried its the wrong direction.

**Pull request closed without merging**